.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Some files here are not required remove them from commit

Reply:
Sure, will do!